elastic/kibana
# [Fleet] Add `installed_kibana_space_id` to `epm-packages` saved objects

## Summary

Part of #74353.

Add the `installed_kibana_space_id` to the `epm-packages` saved object type, and use this field when removing assets. 

This enables us to better handle integrations that have been installed in different spaces, previously if we installed an integration in the default space, then uninstalled it in a different space, the assets would not be deleted.

Manual tests:

- Install a package in the default space
    - check that the `epm-packages` saved object has `installed_kibana_space_id` set to default
    - uninstall the package from a different space
    - check that all assets have been removed.
    
 - Install a package not in the default space e.g space2
    - check that the `epm-packages` saved object has `installed_kibana_space_id` set to space2
    - uninstall the package from a different space e,g default space
    - check that all assets have been removed.
